### PR TITLE
Add helm chart to install eks-sso

### PR DESCRIPTION
**Pull request by nehayward**
# Helm Chart

This adds a fairly simple helm chart to install eks-sso

## Testing
You'll need to create an eks cluster first with helm installed then run the following command with the required values

`helm install eks-sso/ --set sso.name="{name}" --set auth0.connection="{connection}" --set auth0.client_id="{id}" --name eks-sso`

### CODE REVIEW

File: eks-sso/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0.0"
```
Comment (sesh-kebab) [inline eks-sso/Chart.yaml]: [nit] I'd prefer to start at `v0.0.1`

Comment (nehayward) [reply eks-sso/Chart.yaml]: ✅

File: eks-sso/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    k8s-app: eks-sso
+  name: eks-sso
+  namespace: kube-system
+spec:
+  type: LoadBalancer
+  ports:
+    - port: 80
+      targetPort: {{ .Values.sso.port }}
+  selector:
+    k8s-app: eks-sso
```
Comment (sesh-kebab) [inline eks-sso/templates/deployment.yaml]: Selector needs to be updated to `app: eks-sso`.

File: eks-sso/values.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Default values for fluentd-cloudwatch.
```
Comment (sesh-kebab) [inline eks-sso/values.yaml]: copy paste error `fluentd-cloudwatch`

File: eks-sso/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    matchLabels:
+      app: eks-sso
+  template:
+    metadata:
+      labels:
+        app: eks-sso
+    spec:
+      containers:
+      - name: eks-sso
+        image:  "{{ .Values.image.repository }}"
```
Comment (sesh-kebab) [inline eks-sso/templates/deployment.yaml]: Does this need to include a tag as well?

Comment (nehayward) [reply eks-sso/templates/deployment.yaml]: Oops, no so it's just pulling latest. I'll change it to 
`        image:  "{{ .Values.image.repository }}:{{ .Values.image.tag }}"`

Comment (sesh-kebab) [reply eks-sso/templates/deployment.yaml]: Could you make the eks-sso image tag available as a variable from terraform?

**Pull request merged**